scripts/maxmind-db-downloader.js

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var path = require('path');
```

Comment:
Put this in strict mode, I think you'll find there are several variables that miss a `var`